apps/src/feedback.js

```diff
@@ -730,6 +730,12 @@ FeedbackUtils.prototype.getFeedbackMessage = function (options) {
       case TestResults.QUESTION_MARKS_IN_NUMBER_FIELD:
         message = msg.errorQuestionMarksInNumberField();
         break;
+      case TestResults.EXCEEDED_LIMITED_BLOCKS:
+        var exceededBlockType = this.hasExceededLimitedBlocks_();
+        var limit = Blockly.mainBlockSpace.blockSpaceEditor.blockLimits.getLimit(exceededBlockType);
+        var block = `<xml><block type='${this.hasExceededLimitedBlocks_()}'></block></xml>`;
```

Comment:
can be just `type='${exceededBlockType}'`

Reply:
Whoops 😳.  Fixed in ae818e60cc68c60c16f8b98009aa0e28c5aa090c!